javascript/chronometer.js

```diff
@@ -1,34 +1,40 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime=0;
+    this.intervalId=null;
   }
 
   start(callback) {
-    // ... your code goes here
+    const intervalId = setInterval(() => {this.currentTime +=1}, 1000); 
   }
 
   getMinutes() {
-    // ... your code goes here
+    let numberOfMinutes = Math.floor(this.currentTime/60);
+    return numberOfMinutes
   }
 
   getSeconds() {
-    // ... your code goes here
+   let numberOfSeconds = this.currentTime % 60
+   return numberOfSeconds
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    let twoDigitNumber
```

Comment:
if you are not using a variable, you probably should not declare it. Otherwise, clean solution